### ISSUE TITLE
[TF2] Add option to hide spectator bars

### ISSUE THREAD

**Issue by impale1** [ValveSoftware/Source-1-Games#4904]
Black bars make you feel claustrophobic and make it hard to watch the game
![1](https://user-images.githubusercontent.com/117596825/232026083-eaacb4a7-a47f-4558-9256-f8fcb8a81f60.png)

**Comment by kisak-valve** [comment]
Side note, that's not black.

**Comment by Ashetf2** [comment]
I wouldn't like the bars to be removed. They are just too big. Shortening them would be better.

The problem I have with some custom huds that remove the bars is that it can be difficult to read those strings (cycle target, etc.) because TF2 doesn't have the "dropshadow" implementation from more recent Source games (L4D2, CS:GO before Panorama). Someone could just copy an entry in a .res file to duplicate a label and create a fake shadow but it doesn't work in all cases.

**Comment by mtxfellen** [comment]
> I wouldn't like the bars to be removed. They are just too big. Shortening them would be better.

Adding on to this, I suppose there's nothing stopping them from not being bars but rather boxes behind the relevant information, and they could be partially transparent to further clean up screen obscuration.

**Comment by worMatty** [comment]
I agree an option to hide them would be good. But I disagree that they make you feel claustrophobic. I think the widescreen effect can look quite good.

**Comment by CriticalFlaw** [comment]
Simple solution, make them shorter and slightly transparent or make the minmode variant of the spectator view the default.

I don't see the value of adding an option just to remove that part of the HUD.
